internal/naming/annotations.go

```diff
@@ -68,4 +69,10 @@ const (
 	// to a cluster without backups. As usual with the operator, we do not
 	// touch cloud-based backups.
 	AuthorizeBackupRemovalAnnotation = annotationPrefix + "authorizeBackupRemoval"
+
+	// OverrideConfigAnnotation is an annotation used to prevent the controller from reconciling
+	// ConfigMaps when a user wants to manually override their contents. When this annotation
+	// is present, the controller will not update the ConfigMap, allowing users to make custom
+	// modifications that won't be overwritten during reconciliation.
+	OverrideConfigAnnotation = perconaAnnotationPrefix + "override-config"
```

Comment:
Should we mention the ticket here as well?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1108/commits/b3211a04fb5d9cd1c0517df80885c8c372245d48